src/components/main-layout/Layout.tsx

```diff
@@ -17,23 +18,22 @@ interface MainLayoutProps {
 }
 
 const MainLayout: React.FC<MainLayoutProps> = ({
+    layout,
     header,
     leftPanel,
     rightPanel,
     content,
     centerPanel,
     reactionPanel,
 }) => {
-    const { page, exampleTrajectoryPageNumber } = useContext(SimulariumContext);
-
     return (
         <Layout>
             <Header className={styles.navBar}>{header}</Header>
             <Content
                 className={classNames([
                     styles.contentPanel,
                     {
-                        [styles.finalPage]: page === 10,
+                        [styles.finalPage]: layout === LayoutType.FullScreen,
```

Comment:
Nit: are other pages potentially taking this style now? If so maybe `styles.fullScreen`? or something like that